extensions/wikia/VisualEditorTourExperiment/scripts/VisualEditorTourExperiment.js

```diff
@@ -1,14 +1,10 @@
-define('VisualEditorTourExperiment', ['jquery', 'wikia.loader', 'wikia.mustache', 'mw', 'wikia.tracker'],
+define('VisualEditorTourExperiment', ['jquery', 'wikia.loader', 'wikia.mustache', 'mw', 'ext.wikia.spitfires.experiments.tracker'],
 	function ($, loader, mustache, mw, tracker) {
 		'use strict';
 
-		var track = tracker.buildTrackingFunction({
-				category: 've-editing-tour',
-				trackingMethod: 'analytics'
-			});
+		var EXPERIMENT_NAME = 've-tour';
```

Comment:
experiment name should be CONTRIB_EXPERIMENTS as ve-tour in our case is an variation and it's fetched based on experiment name within tracker